get-core.sh

```diff
@@ -59,6 +59,38 @@ pull_docker_image() {
     fi
 }
 
+wait_for_core_to_be_ready() {
+    echo -n "[🔥] Wait for Firebolt Core to be ready"
+    
+    # Try for ~10 seconds to get a valid response from Core
+    timeout=10
+    RESPONSE="Unknown error"
+    while [ $timeout -gt 0 ]; do
+        set +e
+        RESPONSE=$(curl -s 'http://localhost:3473/?output_format=TabSeparatedWithNamesAndTypes' --data-binary "SELECT 42;")
+        set -e
+
+        if [ "$RESPONSE" = $'?column?\nint\n42' ]; then
+            echo " ✅"
+            return 0
+        fi
+        sleep 1
+        timeout=$((timeout - 1))
+        echo -n "."
+    done
+
+    echo " ❌"
+    echo "[❌] Firebolt Core failed to start. This is unexpected, please submit a bug report on Github https://github.com/firebolt-db/firebolt-core/issues"
+    echo "[❌] Error: $RESPONSE"
+    if [ "$(uname)" = "Darwin" ]; then
+        version=$(docker version | sed -n 's/.*Docker Desktop \([0-9.]*\).*/\1/p')
+        if [ "$version" = "4.42.1" ]; then
```

Comment:
No point running Core with this version, the multiple lines of output can be confusing. I'd stop with failure before running Core when Docker Desktop version is 4.42.1.